src/main/java/propertyFiles/astrophysics.properties

```diff
@@ -31,7 +31,7 @@ datasetfieldtype.astroFacility.description=The observatory or facility where the
 datasetfieldtype.astroInstrument.description=The instrument used to collect the data.
 datasetfieldtype.astroObject.description=Astronomical Objects represented in the data (Given as SIMBAD recognizable names preferred).
 datasetfieldtype.resolution.Spatial.description=The spatial (angular) resolution that is typical of the observations, in decimal degrees.
-datasetfieldtype.resolution.Spectral.description=The spectral resolution that is typical of the observations, given as the ratio λ/Δλ.
+datasetfieldtype.resolution.Spectral.description=The spectral resolution that is typical of the observations, given as the ratio \u03bb/\u0394\u03bb.
```

Comment:
This λ/Δλ stuff seems unrelated to #6896. @JayanthyChengan do you mean for this to be in the pull request?

Reply:
It's related to https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/issues/65.  Thought of adding to this PR.

Reply:
@JayanthyChengan thanks. @jggautier and I took a look at the demo server and you're right, this absolutely should be fixed! I'm putting a screenshot below for QA. Should it be fixed in astrophysics.tsv as well?

<img width="476" alt="Screen Shot 2020-05-12 at 2 07 34 PM" src="https://user-images.githubusercontent.com/21006/81743326-772dea80-946f-11ea-887a-5deedfc86c5c.png">